apps/researcher/src/app/[locale]/objects/[id]/date-created-to-string.test.ts

```diff
@@ -0,0 +1,70 @@
+import dateCreatedToString from './date-created-to-string';
+import {describe, expect, it} from '@jest/globals';
+
+describe('dateCreatedToString', () => {
+  it('returns undefined if both startDate and endDate are undefined', () => {
+    const date = dateCreatedToString(
```

Comment:
Minor: could `date` be renamed to make its meaning clear? The name `date` could suggest it's a `Date`, but it's actually a formatted/localized date.